Explain how to create universal binary on macOS (#2405)
scala-native/scala-native: docs/user/sbt.rst

    nativeConfig ~= { _.withTargetTriple("x86_64-apple-macosx10.14.0") }

you may create a few dedicated projects with different target triple.

> For example if I have two `sandbox` project like:

Reviewer: Since lipo is macOS specific we might include this information here
```suggestion
For example, if I have multiple project definitions for different macOS architectures, eg:
```

.. code-block:: scala

    lazy val sandbox64 = project.in(file("sandbox"))
        .settings(nativeConfig ~= { _.withTargetTriple("arm64-apple-darwin20.6.0") })

    lazy val sandboxM1 = project.in(file("sandbox"))
        .settings(nativeConfig ~= { _.withTargetTriple("x86_64-apple-darwin20.6.0") })

This `build.sbt` produces two different binary, one for `x86_64` and another one for `arm64`.

> You may easy to combine it to one so called fat binary or universal binary via

Reviewer: ```suggestion
You may easily combine them to one so called fat binary or universal binary via
```

.. code-block:: sh

>      lipo -create sandbox64/target/scala-2.12/sandbox64-out sandboxM1/target/scala-2.12/sandboxM1-out -output sandbox-out

Reviewer: Some links to the lipo project/reference might be helpful

> which produces `sandbox-out` which can be used at any platform.

Reviewer: ```suggestion
which produces `sandbox-out` that can be used at any platform.
```
Or even better we can include it the sentence before the code block
